lms/djangoapps/philu_api/views.py

```diff
@@ -197,8 +197,9 @@ def get_user_data(request):
 
 def download_pdf_file(request):
     """ Download pdf file (Worksheet) instead of opening in the browser """
-    page_url = request.GET.get("page_url", None)
-    if page_url:
+    query_string = request.META.get('QUERY_STRING')
+    page_url = query_string.split('page_url=')[-1]
```

Comment:
Let's use this library to decode the URL
https://stackoverflow.com/questions/21041787/url-encoding-and-decoding-a-string-in-python